loopback4/src/vulnerabilities/controllerFactory.ts

```diff
@@ -49,6 +51,89 @@ function staticVulnerabilityControllerFactory(
   return [Controller];
 }
 
+function formatParameterSpecs(
+  input: string,
+  params: Param[],
+): ParameterObject[] {
+  if (input === 'body') {
+    return [];
+  }
+
+  // mapping of test-bench-utils parameter types
+  // to lb4 parameter types
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
We were already allowing `any` in at least one other spot - I don't really care to enumerate all of the types myself.